src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -18,6 +18,8 @@ public static partial class Certificates
         {
             private const string CertificatePassword = "testcertificate";
             private const string TestDataFolder = "TestData";
+            private static Mutex m = new Mutex(false, "Global\\CoreFXTest.Configuration.Certificates.LoadPfxCertificate");
+            private const int MutexTimeout = 5 * 1000;
```

Comment:
There's no way to know unless we try it in the CI system... I agree that we'll need to tune this parameter if we see things failing.